src/main/java/org/bitnick/portfolio/service/UserService.java

```diff
@@ -53,10 +73,15 @@ public User updateUser(User user, String id) {
         } catch (Exception ex) {
             LOG.error("An error occurred during update of product" + ex.getMessage());
 
+            // return null if userToUpdate remains null
             return userToUpdate != null ? userToUpdate : new User();
         }
     }
 
+    /**
+     * Deletes a target User from the database.
+     * @param id UUID of target User to be deleted
+     */
```

Comment:
Explain what happens upon the event the the id is invalid. Are those exceptions handled, and how?